prettier/src/trackUsage.ts

```diff
@@ -0,0 +1,29 @@
+const {
+  findConfig,
+  loadConfig,
+  validateConfig,
+  getAccountId,
+  isTrackingAllowed,
+  getAccountConfig,
+} = require("@hubspot/cli-lib");
+import { resolveConfigFile } from "prettier";
+
+export const trackUsage = () => {
+  const rootPath = resolveConfigFile.sync();
```

Comment:
This looks up the prettier config and we use that to try to find the HubSpot config.
It's not guaranteed to be in the a child directly alongside a `hubspot.config.yml` or even exist.